labels/en-US.json

```diff
@@ -754,5 +754,17 @@
   "width": "Width",
   "wrongCountryForType": "Please select a country supported for this contact type",
   "yourAccount": "Your Account",
-  "yourOrderNumber": "Your order number is <strong>{0}</strong>."
+  "yourOrderNumber": "Your order number is <strong>{0}</strong>.",
+  "emailLabel": "Email address(es) (separate emails with a comma)",
+  "emailQuote": "Email Quote",
+  "send": "Send",
+  "confirmDeleteQuote": "Are you sure want to delete {0}?",
```

Comment:
Your labels should be in alphabetical order not all added to the very end. You need to move these labels to the right spot in the file. Alphabetical order.

Fix both files.

Reply:
I have ordered it alphabetically now.